README.md

```diff
@@ -90,7 +92,7 @@ class InvoicePaid extends Notification
 }
 ```
 
-Here's a screenshot preview of the above notification on Facebook Messenger:
+The notification is being sent to the Facebook page you have created before and you will find the message inside the chat window. Here's a screenshot of how this looks:
```

Comment:
I think we need not mention about to whom it is being sent as when they start using either `to()` method or add `routeNotificationForFacebook()` method, then they already know to whom the notification is intended and will be sent to.

So here's a revised version:
> The notification will be sent from your Facebook page whose page token you have configured earlier. Here's a screenshot preview of the notification inside the chat window.